src/app/model/group/player.py

```diff
@@ -8,12 +8,14 @@ class Player(Base):
     """ Tracked player information """
     name: str
     streak: int
-    score: int
+    score: float
+    played: int
 
     @classmethod
     def from_dict(cls, dic):
         return cls(
             name=dic["name"],
             streak=dic["streak"],
-            score=dic["score"]
+            score=dic["score"],
+            played=dic["played"]
         )
```

Comment:
How about `player.games` and `player.score`?

Then we just return the average at presentation time - i.e `player.getAverageScore()` -> `self.score / self.games`

Reply:
I like that approach. Makes it a lot more usable. Good idea